### ISSUE TITLE
add VUS color to cancer types summary

### ISSUE THREAD

**Issue by tmazor** [cBioPortal/cbioportal#8276]
I think it would be great to add an additional color to the Cancer Types Summary plot so that we can distinguish putative driver mutations from VUS -- so instead of a single green bar for mutations, it could be split dark green & light green.

![image](https://user-images.githubusercontent.com/10508276/104623902-dc6e5700-5660-11eb-9d02-d0acee4517db.png)

I think we've probably discussed this before so sorry if it's a duplicate ticket, but it just came up for me so I wanted to raise this again.

**Comment by jjgao** [comment]
Good idea!

I am wondering how we should handle multiple alterations:

- a gene has driver and vus mutations
- a gene has driver CNA and vus mutation
- a gene has driver CNA and driver mutation

Should we simply group them into Multiple?

**Comment by tmazor** [comment]
Oh good question. For the CNA+driver or CNS+VUS, I think those should definitely count as Multiple. For the driver+VUS, I could see counting those as Driver (like how we color samples in OncoPrint, where driver trumps VUS) or as Multiple. I think I would lean towards Multiple since it's a more consistent behavior within this plot.

I wonder if @schultzn has an opinion on this?

**Comment by adamabeshouse** [comment]
@jjgao what is the final product specification for this feature? looks like there is still some discussion

**Comment by jjgao** [comment]
@adamabeshouse based on @tmazor's comment above, let's the following for multiple. 

- a gene has driver and vus mutations => driver
- a gene has driver CNA and vus mutation => multiple
- a gene has driver CNA and driver mutation => multiple

There may be other scenarios and please ask here. Thanks!

**Comment by adamabeshouse** [comment]
So to confirm - we are not distinguishing VUS CNA events from Driver CNA events? And e.g. the amp color will still be the oncoprint driver amp color?

So its 

| CNA \ Mutation | Driver | VUS | None |
| Driver | Multiple | Multiple | CNA event |
| VUS | Mutation driver event | Multiple | CNA Event  |
| None | Mutation driver event | Mutation VUS event | None|

**Comment by jjgao** [comment]
@adamabeshouse I think so. That's also what we decided for OncoPrint https://github.com/cBioPortal/cbioportal/issues/8239#issuecomment-818828403.

**Comment by adamabeshouse** [comment]
oh right, sorry it slipped my mind @jjgao

**Comment by adamabeshouse** [comment]
@jjgao @tmazor how is it handled with driver/vus mutations & cna together with other alterations, i.e. structural variant, mrna, protein?

**Comment by jjgao** [comment]
multiple? @adamabeshouse

**Comment by schultzn** [comment]
I agree with all of these suggestions. And yes, any tumor with more than one event, whether VUS or driver, should just be in "multiple".
